### ISSUE TITLE
replace fs.existsSync in build-server.js

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#337]
Factored this issue out of https://github.com/phetsims/chipper/issues/285#issuecomment-138683623...

Replace uses of deprecated `fs.existsSync` in buildServer.js.   

For cases where there is no fallback and reading the file should fail, simply remove the `fs.existsSync` call. 

For cases where there is a fallback, use this pattern:

```js
try {
   fs.readFileSync( ... );
}
catch ( error ) {
   // do whatever the fallback action is
}
```

**Comment by aaronsamuel137** [comment]
I've updated these usages in build-server. There is one case where I really needed to just check if a file exists, even though there is no need to read it, so I wrote a custom "exists" method using fs.statSync(), which was recommended in the node.js documentation as something to use instead of fs.exists.

I've tested the new code on phet-dev, and seems to be working.

Assigning to @pixelzoom for review.
